### ISSUE TITLE
How to use full screen?

### ISSUE THREAD

**Issue by stuermera** [rsn8887/uae4all2#40]
Hi there!
I tried eery option under "Display" - but somehow I can't get UAE work to use full screen of the PS Vita. There's always a black border left and right...
Any idea?
Thanks.

**Comment by rsn8887** [comment]
That is by design. The Amiga used 4:3 screens. The Vita screen is 16:9. Unless you have a game like Chaos Engine that has a very large black border on the bottom, and zoom in a lot and re-center using “R+Start+Dpad left/right/up/down”, you will always have black borders left and right.

I will try to implement a stretch to full-screen option, but I personally don’t like that. It will distort the games. Circles will become ellipses etc. I don’t think it is how you want to play Amiga games, with such distortion.

**Comment by stuermera** [comment]
Yes I know. It doesn‘t look that good with wrong proportions... but for myself I find it more annoying having those black borders 😂  Perhaps there are others too prefering stretched over black bars.

**Comment by andiweli** [comment]
Any update on this issue? 🤭
